src/pages/form-validation/table/index.tsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import {
+    AppBar,
+    Hidden,
+    IconButton,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { Menu } from '@material-ui/icons';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import * as Colors from '@pxblue/colors';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    toolbarGutters: {
+        padding: '0 16px',
+    },
+    appbarRoot: {
+        padding: 0,
+    },
+    textFieldRoot: {
+        width: 128,
+    },
+    skinnyInput: {
+        paddingTop: 11,
+        textAlign: 'right',
+    },
+    tableContainer: {
+        maxWidth: 650,
+        width: 'auto',
+        marginTop: theme.spacing(4),
+        marginLeft: theme.spacing(2),
+        marginRight: theme.spacing(2),
+        boxSizing: 'border-box',
+    },
+}));
+
+const createData = (id: number, name: string, min: number, max: number): any => ({ id, name, min, max });
+
+const rows = [createData(1, 'Power', 123, 456), createData(2, 'Xpert', 123, 456), createData(3, 'Blue', 123, 456)];
+
+export const TableFormValidation = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const dispatch = useDispatch();
+
+    const getTable = (): JSX.Element => (
+        <TableContainer component={Paper} className={classes.tableContainer}>
+            <Table>
+                <TableHead style={{ fontWeight: 600 }}>
```

Comment:
may want to log an enhancement and make this into theme

Reply:
Good callout, this style actually was already provided as the default.  No issue needed & this inline style was removed.